### ISSUE TITLE
log_url is not set for remote logging

### ISSUE THREAD

**Issue by drmichaelgorski** [fredsa/gwt-log#83]
In my web.xml I do: 

```
    <servlet>   
      <servlet-name>gwt-log-remote-logger-servlet</servlet-name>
      <servlet-class>com.allen_sauer.gwt.log.server.RemoteLoggerServlet</servlet-class>
    </servlet>
    <servlet-mapping>
      <servlet-name>gwt-log-remote-logger-servlet</servlet-name>
      <url-pattern>/gwt-logging</url-pattern>
    </servlet-mapping>
```

On the client side for I do in gwt.xml: 

```
    <set-property name="gwt.logging.logLevel" value="INFO" />
    <set-property name="gwt.logging.enabled" value="TRUE" />
       <set-property name="gwt.logging.simpleRemoteHandler" value="DISABLED" /> 
    <inherits name="com.allen_sauer.gwt.log.gwt-log-ERROR" />
    <extend-property name="log_level" values="DEBUG"/>
    <inherits name="com.allen_sauer.gwt.log.gwt-log-RemoteLogger" />
    <set-property name="log_DivLogger" value="DISABLED" />
    <set-configuration-property name="log_url" value="http://www.example.com/gwt-logging" />
```

But still it tries to log to: `http://www.example.com/gwt-log`.

**Comment by confile** [comment]
@fredsa Do you have any idea on that issue?

**Comment by fredsa** [comment]
Michael,

Your best bet is to read through
https://github.com/fredsa/gwt-log/wiki/GettingStarted#enable-the-remote-logging-option-disabled-by-default

Then, look at the server log to see how the path is being interpreted.

On Thu, Feb 4, 2016 at 5:55 AM, Michael Gorski <notifications@github.com>
wrote:

> @fredsa <https://github.com/fredsa> Do you have any idea on that issue?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/fredsa/gwt-log/issues/83#issuecomment-179850045>.
>



-- 
Fred Sauer
fredsa@gmail.com

**Comment by confile** [comment]
I read this but still I do not see the error. The client tries to log to: ttp://www.example.com/gwt-log

What did I wrong?

**Comment by fredsa** [comment]
You can also check Chrome Dev Tools and look at the network traffic, to
make sure your HTTP request from the client is actually be sent to the
server. Of course, the request should appear in the server request logs as
well. After that, it's a matter of making sure the request is being routed
to the servlet.

On Thu, Feb 4, 2016 at 12:49 PM, Michael Gorski <notifications@github.com>
wrote:

> I read this but still I do not see the error. The client tries to log to:
> ttp://www.example.com/gwt-log
>
> What did I wrong?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/fredsa/gwt-log/issues/83#issuecomment-180040990>.
>



-- 
Fred Sauer
fredsa@gmail.com

**Comment by confile** [comment]
I checked the Chrome Dev Tools of course. The problem is on the client side it still sends the request to 
http://www.example.com/gwt-log no matter what URL I put into 

```
    <set-configuration-property name="log_url" value="http://www.example.com/gwt-logging" />
```

Are you sure that is not a bug?

**Comment by fredsa** [comment]
Always possible of course. Can you just update your web.xml to listed on
/gwt-log instead of /gwt-logging to see if things then work?

On Thu, Feb 4, 2016 at 11:42 PM, Michael Gorski <notifications@github.com>
wrote:

> I checked the Chrome Dev Tools of course. The problem is on the client
> side it still sends the request to
> http://www.example.com/gwt-log no matter what URL I put into
>
>     <set-configuration-property name="log_url" value="http://www.example.com/gwt-logging" />
>
> Are you sure that is not a bug?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/fredsa/gwt-log/issues/83#issuecomment-180238897>.
>



-- 
Fred Sauer
fredsa@gmail.com

**Comment by confile** [comment]
@fredsa Yes it work with /gwt-logthe problem still exist that I cannot change the path on the client side. 

Do you have any idea or workaround for this?
Thanks for help.

**Comment by fredsa** [comment]
There's an example working configuration in the client server demo:

https://github.com/fredsa/gwt-log/blob/master/Log/clientserverdemo/com/allen_sauer/gwt/log/clientserverdemo/LogClientServerDemo.gwt.xml

That's the code for the demo: http://gwt-log.appspot.com/

On Sat, Feb 6, 2016 at 5:32 AM, Michael Gorski <notifications@github.com>
wrote:

> @fredsa <https://github.com/fredsa> Yes it work with /gwt-logthe problem
> still exist that I cannot change the path on the client side.
>
> Do you have any idea or workaround for this?
> Thanks for help.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/fredsa/gwt-log/issues/83#issuecomment-180765449>.
>



-- 
Fred Sauer
fredsa@gmail.com
